### PR TITLE
Feat: added Modal

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    padding: 0.75rem 1rem;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+    width: fit-content;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```
Comment (Aman-Jain1996) [inline components/modal/modal.css:12]: Just a recommendation rohan , if you are using flex try to use it to center the modal not position's

**Review by Aman-Jain1996 — APPROVED** [review]
Looks good to go

**Review by hrishi82 — COMMENTED** [review]
Looks good to merge.

Suggestion: Since it's a modal, I believe it should take up the whole viewport. So, I guess position fixed will be a better option to use.

File: components/modal/modal.css (hunk 1 above)
Comment (Rohanmond) [reply components/modal/modal.css:12]: we need modal on top of the other elements so for that I think justify content or align items property will not work...I should use position as fixed and fixed that now. Thanks for your review. You can check now.

**Comment by Rohanmond** [comment]
> Looks good to merge.
> 
> Suggestion: Since it's a modal, I believe it should take up the whole viewport. So, I guess position fixed will be a better option to use.

I thought of that but it will give me a problem while documentation so I gave position as absolute. Thanks for your review.

**Pull request merged**